.ci/scripts/dswx_compare_opera_pge.py

```diff
@@ -0,0 +1,221 @@
+from osgeo import gdal
+import numpy as np
+import os
+import sys
+
+COMPARE_DSWX_HLS_PRODUCTS_ERROR_TOLERANCE_ATOL = 1e-6
+COMPARE_DSWX_HLS_PRODUCTS_ERROR_TOLERANCE_RTOL = 1e-5
+
+def _get_prefix_str(flag_same, flag_all_ok):
```

Comment:
I think this was also in the original script.  I don't like changing input values as a surrogate for a return value, and would rather see a value for `flag_all_ok` returned instead.  But I cede to whatever the convention is for this repo.